scripts/tasks/sandbox.ts

```diff
@@ -55,9 +55,10 @@ export const sandbox: Task = {
       await remove(details.sandboxDir);
     }
 
-    const { create, install, addStories, extendMain, init, addExtraDependencies } = await import(
-      './sandbox-parts'
-    );
+    const { create, install, addStories, extendMain, init, addExtraDependencies } =
+      // @ts-expect-error esbuild for some reason exports a default object
+      // eslint-disable-next-line import/extensions
+      (await import('./sandbox-parts.ts')).default;
```

Comment:
Does anyone have an idea why I have to import `default`, instead of accessing the exported functions directly?

Reply:
What's stopping us from importing this top-level with a regular import? It can't be _that_ much slower.

Reply:
I tried it, but for some reason, the import is runtime-sensitive and only works if some preconditions are fulfilled before.